bin/node-template/node/src/cli.rs

```diff
@@ -10,6 +10,7 @@ pub struct Cli {
 }
 
 #[derive(Debug, clap::Subcommand)]
+#[allow(clippy::large_enum_variant)]
```

Comment:
The CI clippy check is already green: https://gitlab.parity.io/parity/mirrors/substrate/-/jobs/2656259  
I dont know what local config you are using, but just adding `allow`s here will be guaranteed to get outdated, since it is not CI enforced.

Reply:
Looks like the CI uses `SKIP_WASM_BUILD=1 env -u RUSTFLAGS cargo +nightly clippy --all-targets` with `1.67.0-nightly (a3dfea71c 2022-11-11)`.

Reply:
I don't understand why the allow annotation would become outdated (perhaps because I don't know much about your CI).

The only reason I see for it becoming outdated is if the clippy label was renamed, which does not seem very likely.

I ran into this using version 1.68.1 of rustc and clippy, btw.